docs/topics/rn-known-issues.adoc

```diff
@@ -17,23 +17,27 @@ For a complete list of all known issues, see the list of link:https://issues.red
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
-|CodeReady Workspace plugin
-|The plugin works with CodeReady Workspace 2.09, but not version 2.10.
+|link:https://issues.redhat.com/browse/WINDUP-3227[WINDUP-3227]
+|Migration - Application
+|The `IncludeTags` advanced option does not work.
 
-|link:https://issues.redhat.com/browse/WINDUP-3165[WINDUP-3165]
-|Web console on OpenShift Container Platform
-|Analysis fails when MTA custom labels are uploaded and advanced options are enabled.
+|link:https://issues.redhat.com/browse/WINDUP-3266[WINDUP-3266]
+|IntelliJ extension
+|An exception related to `getFile` intermittently occurs when running an analysis via the extension.
 
-|link:https://issues.redhat.com/browse/WINDUP-3160[WINDUP-3160]
+|link:https://issues.redhat.com/browse/WINDUP-3267[WINDUP-3267]
+|Web UI & Windup as a service (`windup-web`)
+|There is no option in the web console to navigate to or to download mavenized `pom.xml` files generated if you enable the `mavenized` option.
+
+|link:https://issues.redhat.com/browse/WINDUP-3318[WINDUP-3318]
 |Web console on OpenShift Container Platform
-|If you run an analysis on multiple applications and subsequently delete an application and rerun the analysis, the analysis hangs.
+|Analysis fails when MTA custom labels are uploaded and advanced options are enabled.
 
-|link:https://issues.redhat.com/browse/WINDUP-3005[WINDUP-3005]
-|Web console
-|The *Save* and *Save and run* buttons are not displayed on the *Custom rules* and *Custom labels* tabs on the *Analysis configuration* page.
+|link:https://issues.redhat.com/browse/WINDUP-3322[WINDUP-3322]
+|Web UI & Windup as a service (`windup-web`)
+|When you edit any project and make any changes to a custom rule or a custom label, the *Save* and *Save and Run* buttons remain disabled. Workaround: You can make these changes in the *Analysis configuration* screen.
 
```

Comment:
@RichardHoch 
Web UI

Reply:
@sbeskin-redhat -- Making the change but it does mean that the description here of the component does not match the description of the component in the Jira I reference. Something to think about later.